src/pages/[platform]/build-a-backend/add-aws-services/analytics/set-up-analytics/index.mdx

```diff
@@ -378,6 +378,10 @@ To initialize the Amplify Auth and Analytics categories you call `Amplify.addPlu
 
 Add the following code to your `onCreate()` method in your application class:
 
+<Callout warning>
+Before calling the `Amplify.configure` function, make sure to move the `amplify_outputs.json` file you generated into the `app/src/main/res/raw` directory in your Android project. Otherwise, you will not be able to connect to your backend.
```

Comment:
The `app/src/main/res/raw` path is project-dependent. I would perhaps say "into the appropriate resource directory for your application (for example, `app/src/main/res/raw`)".

The "Otherwise, you will not be able to connect to your backend." line is also a bit incorrect - if the file is not present you will not be able to compile your application because `R.raw.amplify_outputs` will not exist.

Finally, do we want to note that the file can be placed here using the output directory flag on the CLI?

Reply:
> do we want to note that the file can be placed here using the output directory flag on the CLI?

I think we should call that out here. There are mentions in other pages that we can copy/paste from

Reply:
I approved but I think we also need to educate the customer on installing `npx amplify` as we can't assume a native front end developer has node installed, npx installed, and is also aware of how to install amplify package.